readme.md

```diff
@@ -1,46 +1,105 @@
-anyone can write their own, which is intended to provide natural
-growth of tools within this framework.
+[![Binder Badge](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/NREL/OpenOA/main?filepath=examples)
+[![Gitter Badge](https://badges.gitter.im/NREL_OpenOA/community.svg)](https://gitter.im/NREL_OpenOA/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Journal of Open Source Software Badge](https://joss.theoj.org/papers/d635ef3c3784d49f6e81e07a0b35ff6b/status.svg)](https://joss.theoj.org/papers/d635ef3c3784d49f6e81e07a0b35ff6b)
 
-The library is written around Pandas Data Frames, utilizing a flexible backend
-so that data loading, processing, and analysis could be performed using other libraries,
-such as Dask and Spark, in the future.
+[![Documentation Badge](https://readthedocs.org/projects/openoa/badge/?version=latest)](https://openoa.readthedocs.io)
+![Tests Badge](https://github.com/NREL/OpenOA/workflows/Tests/badge.svg?branch=develop)
+[![Code Coverage Badge](https://codecov.io/gh/NREL/OpenOA/branch/develop/graph/badge.svg)](https://codecov.io/gh/NREL/OpenOA)
 
-If you would like to try out the code before installation or simply explore the possibilities, please see our examples on [Binder](https://mybinder.org/v2/gh/NREL/OpenOA/develop_v3?filepath=examples).
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Imports: isort](https://img.shields.io/badge/%20imports-isort-%231674b1?style=flat&labelColor=ef8336)](https://pycqa.github.io/isort/)
 
-If you use this software in your work, please cite our JOSS article with the following BibTex:
+-----
 
-```
-@article{Perr-Sauer2021,
-  doi = {10.21105/joss.02171},
-  url = {https://doi.org/10.21105/joss.02171},
-  year = {2021},
-  publisher = {The Open Journal},
-  volume = {6},
-  number = {58},
-  pages = {2171},
-  author = {Jordan Perr-Sauer and Mike Optis and Jason M. Fields and Nicola Bodini and Joseph C.Y. Lee and Austin Todd and Eric Simley and Robert Hammond and Caleb Phillips and Monte Lunacek and Travis Kemper and Lindy Williams and Anna Craig and Nathan Agarwal and Shawn Sheng and John Meissner},
-  title = {OpenOA: An Open-Source Codebase For Operational Analysis of Wind Farms},
-  journal = {Journal of Open Source Software}
-}
+# Overview
+
+- Jump to [installation instructions](#installation-and-usage).
+- Try out the Binder [demo notebook](https://mybinder.org/v2/gh/NREL/OpenOA/main?filepath=examples).
+- Read the [documentation](https://openoa.readthedocs.io/en/latest/).
+- Learn how to [contribute](contributing.md).
+
+This library provides a framework to study operational assessment (OA) methodologies that consume
+timeseries data from wind plants. Its development has been motivated by the National Renewable
```

Comment:
To make it clear how OpenOA fits into the WP3 benchmark project, I suggest we say: "Its development has been motivated by the Wind Plant Performance Prediction (WP3) Benchmark project, led by the National Renewable Energy Laboratory (NREL), which focuses on quantifying and understanding differences between the expected and actual energy production of wind plants. To support the WP3 Benchmark project, OpenOA was initially developed to provide a baseline implementation of a long-term operational annual energy production (AEP) estimation method. It has since grown to incorporate several more OA analysis methods, lower-level utility functions, and a schema…"